CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Bugfixes
+
+- Fixed an issue where `util.clean_headers()` would throw a `ValueError` if a user accidentally included a space in their API token. (Thanks, [Keegan](https://github.com/keeeeeegan))
```

Comment:
Generally, we use `@username` instead of their name in the changelog.

Change to `@keeeeeegan`?